qctrlopencontrols/dynamic_decoupling_sequences/dynamic_decoupling_sequence.py

```diff
@@ -39,29 +39,28 @@
 
 
 class DynamicDecouplingSequence:
-    """
+    r"""
     Create a dynamic decoupling sequence.
-    Can be made of perfect operations, or realistic pulses.
 
     Parameters
     ----------
     duration : float
         Defaults to 1. The total time in seconds for the sequence.
     offsets : np.ndarray
         Defaults to None.
-        The times offsets in s for the center of pulses.
+        The times offsets :math:`\{t_j\}` in seconds for the center of pulses.
         If None, defaults to one operation at halfway [0.5].
     rabi_rotations : np.ndarray
         Defaults to None.
-        The rabi rotations at each time offset.
-        If None, defaults to np.pi at each time offset.
+        The rabi rotation :math:`\omega_j` at each time offset :math:`t_j`.
```

Comment:
Should this be in braces too ({omega_j})? (same for the others)

Reply:
I was thinking it's defined for each time offset.

Reply:
Ah, makes sense.